.github/workflows/release.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - name: Build
         run: hatchling build
       - name: Push to PyPI
-        uses: pypa/gh-action-pypi-publish@v1.9.0
+        uses: pypa/gh-action-pypi-publish@v1.10.0
```

Comment:
@dbanty please bump to v1.10.1 as I fixed a minor bug. It does not affect you yet, since it's hit by the attestations code path. However, I invite you to enable this new experimental feature and try it out.

Reply:
Bumped and enabled in https://github.com/openapi-generators/openapi-python-client/pull/1116